docs/source/getting_started/installation.rst

```diff
@@ -85,3 +85,9 @@ You can also build and install vLLM from source:
 
         $ nvcc --version # verify that nvcc is in your PATH
         $ ${CUDA_HOME}/bin/nvcc --version # verify that nvcc is in your CUDA_HOME
+
+    Sometimes, forcing a reinstall can help resolve build errors.
+
+    .. code-block:: console
+
+        $ pip install -e . --no-cache-dir --force-reinstall
```

Comment:
`--no-cache-dir` is not good. This means we need to download all packages again. only `--force-reinstall` is enough.

Reply:
Sorry, I think this is actually the right command:

```sh
 `CUDACXX=/usr/local/cuda-12/bin/nvcc pip install -e  .`
```

Reply:
Yeah it seems like nvcc just wasn't in my path, which the docs did mention. I guess I just didn't understand. My bad. Going to cancel the PR.